### PR TITLE
[4.0] com_cpanel. Add menu " Modules"

### PR DESCRIPTION
**Pull request by ReLater**
### Summary of Changes
- Adds menu "Modules" to "System" > "Manage" in BE.

### Testing Instructions
- See also images below.
- Joomla4 nightly build.
- Goto menu "System".
- "Modules" not present in category "Manage"
- Apply patch.
- "Modules" present in category "Manage"

### CODE REVIEW

**Comment by brianteeman** [comment]
Rightly or wrongly the modules can be managed using the link present in the content sub menu

**Comment by ReLater** [comment]
Does it disturb anyone to have it here, too? I just know that some guys are nagging because searching for menus in new BE is really "a pest". I myself would expect "Modules" under "Manage".

Without your hint now I haven't found "Modules" and always used an `option=com_modules` link in address bar. "Modules" never was a Submenu of "Content" in the past.

Let's do it like this: I don't close here and one of the "company bosses" can close whenever wanted. I disagree but will bite my lip.

**Comment by brianteeman** [comment]
Just pointing out that it was a deliberate decision to move modules to content. Not stating if I agree or not. But if the argument is that it was never there before then that's a poor argument and we might as well never change or improve.

**Comment by ReLater** [comment]
> But if the argument is that it was never there before then that's a poor argument and we might as well never change or improve.

The basic main argument is 

> I just know that some guys are nagging because searching for menus in new BE is really "a pest". I myself would expect "Modules" under "Manage".

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 07289e8163845e814189fce9fdc89157e98b6d41<br />I find Modules under Manage more user friendly! Thank you. <br /><hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22132">issues.joomla.org/tracker/joomla-cms/22132</a>.</sub>

**Comment by ChristineWk** [comment]
I find Modules under Manage more user friendly! Thank you. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22132">issues.joomla.org/tracker/joomla-cms/22132</a>.</sub>

File: administrator/language/en-GB/en-GB.mod_menu.ini [hunk 1]
```diff
@@ -121,6 +121,7 @@ MOD_MENU_MANAGE_LANGUAGES="Language(s)"
 MOD_MENU_MANAGE_LANGUAGES_CONTENT="Content Language(s)"
 MOD_MENU_MANAGE_LANGUAGES_OVERRIDES="Language Overrides"
 MOD_MENU_MANAGE_PLUGINS="Plugins"
+MOD_MENU_MANAGE_MODULES="Modules"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.mod_menu.ini]: Sort in alpha order.

**Review by brianteeman — APPROVED** [review]
approve language change

**Comment by chmst** [comment]
Modules are content and placed on a prominent place as quickicon in the dashboard. In theory this is good after a short time I am used to this.

But: Modules also are extensions, they are kind of hybrid extensions. They belong as well to templates, as they may have a template position. 

They are extension which must be installed as other extensions. 

If I work on a template, then maybe I want to change the position or chrome or bootstrap-size of a module - and at the moment I must return to the dashboard to do it. 

My2cent: When a user is on the sytems screen, he expects to find modules here too, together with plugins and other add-ons. Does it make a harm to have an additional link to modules in the manage-box and on dashboard? 
For me, I would appreciate an extra box for all the language stuff (three items). Then "manage" container would be fine with modules above the plugins. The link to modules can be also kept in the dashboard as it belongs also to content.


<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22132">issues.joomla.org/tracker/joomla-cms/22132</a>.</sub>

**Pull request closed without merging**